git_services/git_services/sidecar/rpc_methods.py

```diff
@@ -0,0 +1,108 @@
+"""RPC Server methods.
+
+Note that all __doc__ of functions below are templated into a <pre></pre> section
```

Comment:
There's no `__doc__` below.

Reply:
Oh my bad that is leftover from when there was __doc__.